specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/quotas.json

```diff
@@ -0,0 +1,271 @@
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The container registry quota has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryQuota"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/quotas": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of container registry quotas at the given location.": {
+            "$ref": "./examples/Quotas/List.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryQuotas"
+        ],
+        "description": "Returns a list of container registry quotas at the given location.",
+        "operationId": "ContainerRegistryQuotas_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of container registry quotas has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryQuotaList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryQuota": {
+      "description": "Container registry quota.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry quota properties.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryQuotaProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
+        }
+      ]
+    },
+    "ContainerRegistryQuotaProperties": {
+      "description": "Container registry quota properties.",
+      "type": "object",
+      "properties": {
+        "numberOfRegistries": {
+          "description": "Total number of container registry accounts.",
+          "type": "integer",
+          "format": "int32",
+          "default": 20
```

Comment:
note that default specifies the value that will occur on the service if no value is specified.

Reply:
It is reasonable default value

Reply:
The default value here specifies the one on the RP side.
The generated client will not supply the specified value here as default, thats what mark is saying . Is that fine?

Reply:
I think it is ok

Reply:
yes it should be fine.